hack/verify-golint.sh

```diff
@@ -57,28 +57,25 @@ export IFS=$'\n'
 all_packages=(
 	$(go list -e ./... | egrep -v "/(third_party|vendor|staging/src/k8s.io/client-go/pkg|generated|clientset_generated)" | sed -e 's|^k8s.io/kubernetes/||' -e "s|^_${KUBE_ROOT}/\?||")
 )
-linted_packages=(
-	$(cat $linted_file)
+failing_packages=(
+	$(cat $failure_file)
```

Comment:
I wonder if perhaps we should `| awk '{print $1}'` or something when we cat the file in, so that we could possibly add comments to the failure file.. have some sort of trail of why the failure exists, or a PR/Issue to look at.